stockflux-securities-master/src/components/InputForm.js

```diff
@@ -0,0 +1,101 @@
+import React, { useRef } from "react";
+import { Link } from "react-router-dom";
+import "./InputForm.css";
+
+const InputForm = () => {
+  const nameRef = useRef(null);
```

Comment:
The [official recommendation regarding refs](https://reactjs.org/docs/refs-and-the-dom.html) is to avoid them where possible. Instead of using refs, you should use state to manage the data flow between DOM elements and the local component state. See [here](https://reactjs.org/docs/forms.html) for examples on how to do this, and [here](https://reactjs.org/docs/hooks-state.html) for documentation on how to manage state using the state hook.

Reply:
Okay I've made the change now